pkg/Microsoft.Private.CoreFx.UAP/Microsoft.Private.CoreFx.UAP.pkgproj

```diff
@@ -35,6 +35,8 @@
     <ExcludeFromClosure Include="System.Net" />
     <ExcludeFromClosure Include="System.Transactions" />
 
+    <ExcludeFromDuplicateTypes Include="System.Private.Reflection.Metadata.Ecma335" />
```

Comment:
Feel free to file an issue for that if necessary.  I wasn't sure of that because I suspect someone needs to call these and we don't tend to use InternalsVisibleTo.